src/test/datascience/nativeEditor.functional.test.tsx

```diff
@@ -482,77 +484,79 @@ suite('DataScience Native Editor', () => {
                     }
                 });
 
-                runMountedTest('Remote kernel can be switched and remembered', async () => {
+                runMountedTest('Remote kernel can be switched and remembered', async function () {
                     // Turn off raw kernel for this test as it's testing remote
                     ioc.forceDataScienceSettingsChanged({ disableZMQSupport: true });
 
                     const pythonService = await createPythonService(ioc, 2);
 
-                    // Skip test for older python and raw kernel and mac
-                    if (pythonService && os.platform() !== 'darwin' && !ioc.mockJupyter) {
-                        const uri = await startRemoteServer(ioc, pythonService, [
-                            '-m',
-                            'jupyter',
-                            'notebook',
-                            '--NotebookApp.open_browser=False',
-                            '--NotebookApp.ip=*',
-                            '--NotebookApp.port=9999'
-                        ]);
-
-                        // Set this as the URI to use when connecting
-                        await ioc.setServerUri(uri);
-
-                        // Create a notebook and run a cell.
-                        const notebook = await createNewEditor(ioc);
-                        await addCell(notebook.mount, 'a=12\na', true);
-                        verifyHtmlOnCell(notebook.mount.wrapper, 'NativeCell', '12', CellPosition.Last);
-
-                        // Create another notebook and connect it to the already running kernel of the other one
-                        when(ioc.applicationShell.showQuickPick(anything(), anything(), anything())).thenCall(
-                            async (o: IKernelSpecQuickPickItem[]) => {
-                                const existing = o.filter(
-                                    (s) =>
-                                        s.selection.kind === 'connectToLiveKernel' &&
-                                        s.selection.kernelModel.numberOfConnections
-                                );
+                    // Skip test for older python and raw kernel
+                    if (!pythonService || ioc.mockJupyter){
+                        return this.skip();
```

Comment:
Used `skip` instead of conditionally not running a test.
I was running the tests locally and spent a lot of time trying to figure out why it was workign locally & not on the server, as locally mocha results indicated it ran and completed successfully.

Found that we had if conditions that didn't use `skip`, which produces different outputs.